src/tools/radosacl.cc

```diff
@@ -151,7 +151,7 @@ int main(int argc, const char **argv)
 
   ACLID id;
 
-  snprintf(id.id, ID_SIZE + 1, "%.16x", 0x1234);
+  snprintf(id.id, ID_SIZE + 1, "%.8x", 0x1234);
```

Comment:
as i replied you over IRC. i was suggesting replace `ID_SIZE` with `sizeof(id.id)`, in other words:
```c++
snprintf(id.id, sizeof(id.id), "%.8x", 0x1234);
```